src/wp-includes/post.php

```diff
@@ -6450,13 +6451,14 @@ function wp_delete_attachment( $post_id, $force_delete = false ) {
  *
  * @global wpdb $wpdb WordPress database abstraction object.
  *
- * @param int    $post_id      Attachment ID.
- * @param array  $meta         The attachment's meta data.
- * @param array  $backup_sizes The meta data for the attachment's backup images.
- * @param string $file         Absolute path to the attachment's file.
+ * @param int    $post_id        Attachment ID.
+ * @param array  $meta           The attachment's meta data.
+ * @param array  $backup_sizes   The meta data for the attachment's backup images.
+ * @param string $file           Absolute path to the attachment's file.
+ * @param array  $backup_sources The meta data for the attachment's backup sources images.
  * @return bool True on success, false on failure.
  */
-function wp_delete_attachment_files( $post_id, $meta, $backup_sizes, $file ) {
+function wp_delete_attachment_files( $post_id, $meta, $backup_sizes, $file, $backup_sources ) {
```

Comment:
As per the WordPress document, we need to add a `@since` document for the new variable.

`* @since 6.1.0 The $backup_sources parameter was added.`

Reply:
Added the PHPDoc. thanks 👍